src/main/java/net/amygdalum/tanteemmas/external/TimeProvider.java

```diff
@@ -21,29 +23,30 @@ public void setSpeed(long speed) {
 		this.start = current - newMillis;
 		this.speed = speed;
 	}
-	
+
 	public long getSpeed() {
 		return speed;
 	}
 
+	@Input
```

Comment:
Isn't this commit supposed to only format code? This is a annotation from Stefan's tool (also below in `PriceCalculator` and elsewhere).

Moreover, what formatter settings are you using? Because there's no `.settings/` folder in the repo.

Reply:
Fixed.